tensorflow/core/kernels/bincount_op.cc

```diff
@@ -182,6 +183,12 @@ struct BincountReduceFunctor<CPUDevice, Tidx, T, binary_output> {
           for (int64_t i = start_row; i < end_row; ++i) {
             for (int64_t j = 0; j < num_cols; ++j) {
               Tidx value = in(i, j);
+              if (value < 0) {
+                status.Update(errors::InvalidArgument(
```

Comment:
This isn't thread-safe.  We may need an initial `in.minimum()` reduction (or `argmin() if you want the bad input index) prior to launching the parallel-for.